ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/common/validation/Validators.kt

```diff
@@ -96,6 +101,22 @@ fun <E> Validator<E>.Property<String?>.isUrl() =
         }
     }.hasMaxLengthOf(255)
 
+/**
+ * Valid if the string is a URL following the w3.org spec
+ */
+fun String?.isUrl() =
+    validate(this) {
+        it == null || isUrl(it)
+    }
```

Comment:
You should rename it `validateUrl()` so that it implicitly states that it throws an exception if it's not a URL